### ISSUE TITLE
A11y_Azure Data Studio_Toolbar_Properties_Keyboard: The tooltips in the table are not keyboard accessible.

### ISSUE THREAD

**Issue by Truptik12** [microsoft/azuredatastudio#20785]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #Keyboard; #WCAG2.1.1; #FTP;

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:

1. Launch the "Azure data studio" application.
2. Connect with the provided "sqltools2019-3" SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. TAB to the toolbar present at right side of the screen.
7. TAB to "Properties" button and hit ENTER key.
8. Tab through the table and observe that the tooltips are not keyboard accessible.

### Actual:
The tooltips in the table are not keyboard accessible..

**This issue is observed throughout the application**

### Expected Result:
The tooltips in the table should be keyboard accessible..

### Attachment:
![Screen Recording (8)](https://user-images.githubusercontent.com/86710491/194823683-87a20af0-923f-4a77-85ad-edf447de9619.gif)

**Comment by alanrenmsft** [comment]
@Truptik12 
dupe of: https://github.com/microsoft/azuredatastudio/issues/19566
we've already have a sev3 issue for AzureDataStudio.
please close the issue.
